lib/rules/custom-property-pattern/__tests__/index.js

```diff
@@ -19,6 +19,12 @@ testRule({
 		{
 			code: ':root { --foo-sub-color: #f00; } a { color: var(--foo-color, var(--foo-sub-color)); }',
 		},
+		{
+			code: 'a { color: var($foo-color); }',
+		},
+		{
+			code: 'a { color: var(tokens.$foo-color); }',
+		},
```

Comment:
Let's move this into a separate `testRule` that sets the `customSyntax` property ([like so](https://github.com/stylelint/stylelint/blob/63b22e065c5e7fbf1e25e9626b167918b81589cd/lib/rules/alpha-value-notation/__tests__/index.js#L324-L357)) as we like to keep non-standard syntax separate from standard syntax.